pkg/doc/signature/signer/signer.go

```diff
@@ -121,7 +121,7 @@ func (signer *DocumentSigner) signObject(context *Context, jsonLdObject map[stri
 		p.JWS = proof.CreateDetachedJWTHeader(p) + ".."
 	}
 
-	message, err := proof.CreateVerifyData(suite, jsonLdObject, p, jsonld.WithRemoveAllInvalidRDF())
+	message, err := proof.CreateVerifyData(suite, jsonLdObject, p, jsonld.WithValidateRDF())
```

Comment:
`WithValidateRDF` option is for jsonld consumers. Should this option be available to consumers of presentation/credential packages as well ?

Reply:
it is already available as remove all RDF, if provided then this option will not be used